Workbooks/Workspace/Agent Health/Agent Health.workbook

```diff
@@ -104,10 +172,10 @@
     {
       "type": 1,
       "content": {
-        "json": "<br>\r\n### ⚠️ No agent health data found\r\n---\r\n\r\nLooks like no Log Analytics agent is sending health data to this workspace.\r\n\r\nPlease check out the detailed documentation on how to enable Log Analytics agent for your environment [here](https://docs.microsoft.com/azure/azure-monitor/platform/log-analytics-agent)."
+        "json": "<br>\r\n### ⚠️ No agent health data found\r\n---\r\n\r\nLooks like no Log Analytics agent is sending health data to this workspace.\r\n\r\nPlease select another workspace or check out the detailed documentation on how to enable Log Analytics agent for your environment [here](https://docs.microsoft.com/azure/azure-monitor/platform/log-analytics-agent)."
       },
       "conditionalVisibility": {
-        "parameterName": "HasHeartbeat",
+        "parameterName": "HasHearbeat",
```

Comment:
need to change all the references here also

Reply:
Search/replace and found all other refs and updated.